README.rst

```diff
@@ -40,14 +40,59 @@ USAGE
 
 If you want to learn more about coala-bears, its functionality and its usage,
 please take a look at our
-`tutorial <http://coala.readthedocs.org/en/latest/Users/Tutorials/Writing_Bears.html#what-is-a-bear>`__.
+`tutorial <http://coala.readthedocs.org/en/latest/index.html>`__.
 
 ABOUT
 =====
 
-coala-bears is a Python package containing all the bears that is used by coala.
-
-Read more at our `documentation <http://coala.readthedocs.org/en/latest/Users/Tutorials/Writing_Bears.html#guide-to-write-a-bear>`__.
+coala-bears is a Python package containing all the bears that are used by coala.
+It has been split from `coala <https://github.com/coala-analyzer/coala>`_.
+With the initial release, it features 56 bears covering 32 languages.
+You can see all of them `here <https://gist.github.com/Adrianzatreanu/cf2d0c8b2ecd542a4860>`_
+with a brief description each.
+
+coala-bears has bears for famous languages, such as:
```

Comment:
Possibly put "53 languages" in bold ?

On Wed, Mar 30, 2016 at 1:53 PM, Attila <notifications@github.com> wrote:

> In README.rst
> <https://github.com/coala-analyzer/coala-bears/pull/300#discussion_r57851966>
> :
>
> >
> >  ABOUT
> >  =====
> >
> > -coala-bears is a Python package containing all the bears that is used by coala.
> > -
> > -Read more at our `documentation <http://coala.readthedocs.org/en/latest/Users/Tutorials/Writing_Bears.html#guide-to-write-a-bear>`__.
> > +coala-bears is a Python package containing all the bears that are used by coala.
> > +It has been split from `coala <https://github.com/coala-analyzer/coala>`_.
> > +With the initial release, it features 56 bears covering 32 languages.
> > +You can see all of them `here <https://gist.github.com/Adrianzatreanu/cf2d0c8b2ecd542a4860>`_
> > +with a brief description each.
> > +
> > +coala-bears has bears for famous languages, such as:
>
> I think this is rather essential information, more like promotional. If we
> don't convince them right away, that we worth a time, no one will read the
> wiki page.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/coala-analyzer/coala-bears/pull/300/files/1c930c1e38103799b7c549fcfe2e29b210db8494#r57851966>
>